scenario/GovernanceScenario.ts

```diff
@@ -313,7 +313,8 @@ async function fastL1ToPolygonGovernanceExecute(
     throw new Error('cannot execute governance without governance deployment manager');
   }
 
-  const proposer = await impersonateAddress(governanceDeploymentManager, COMP_WHALES.testnet[0]);
+  const compWhale = world.base.network === 'polygon' ? COMP_WHALES.mainnet[0] : COMP_WHALES.testnet[0];
```

Comment:
I think we should be able to use `ctx.getProposer()` here.

We basically mimic this logic in `getProposer`:

https://github.com/compound-finance/comet/pull/672/files#diff-a1090d669b0fd5457f94875e225762c4326775adfa733c3bfa9f8689ffa2c24dR70

Reply:
Actually, I don't think we can because we need to use the `governanceDeploymentManager` here while `ctx.getProposer()` uses the base `DeploymentManager`.

Reply:
aha, because of `impersonateAddress`

Reply:
yep